options/locale/locale_en-US.ini

```
@@ -249,7 +249,7 @@ no_reply_address = Hidden Email Domain
 no_reply_address_helper = Domain name for users with a hidden email address. For example, the username 'joe' will be logged in Git as 'joe@noreply.example.org' if the hidden email domain is set to 'noreply.example.org'.
 password_algorithm = Password Hash Algorithm
 invalid_password_algorithm = Invalid password hash algorithm
-password_algorithm_helper = Set the password hashing algorithm. Algorithms have differing requirements and strength. `argon2` whilst having good characteristics uses a lot of memory and may be inappropriate for small systems.
+password_algorithm_helper = Set the password hashing algorithm. Algorithms have differing requirements and strength. The argon2 whilst having good characteristics uses a lot of memory and may be inappropriate for small systems.
```

```suggestion
password_algorithm_helper = Set the password hashing algorithm. Algorithms have differing requirements and strength. The argon2 algorithm is the most secure but uses a lot of memory and may be inappropriate for small systems.
```

---

"most secure" would cause arguments.

The purpose of this change it just to re-trigger the translation, I don't want to change too much.

---

```suggestion
password_algorithm_helper = Set the password hashing algorithm. Algorithms have differing requirements and strength. The argon2 algorithm is rather secure but uses a lot of memory and may be inappropriate for small systems.
```
What about that instead? The problem is that what you're proposing isn't English.

---

> The problem is that what you're proposing isn't English.

That's not written by me , actually .....

---

(The previous one was, though)

---

LOL,  I am not English native speaker, so I dare not change what other people wrote too much 😂